ivy/functional/frontends/numpy/linalg/matrix_and_vector_products.py

```diff
@@ -57,6 +57,13 @@ def einsum(
     return ivy.einsum(subscripts, *operands, out=out)
 
 
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def gradient(a, /):
+    a = promote_types_of_numpy_inputs(a)
+    return ivy.gradient(a)
```

Comment:
Please look into the documentation of [ivy.gradient](https://unify.ai/docs/ivy/docs/functional/ivy/experimental/elementwise/ivy.functional.ivy.experimental.elementwise.gradient.html#gradient) to understand the different parameters it takes as input.